### ISSUE TITLE
Pillager bounding box incorrect

### ISSUE THREAD

**Issue by tomosmc** [maruohon/minihud#265]
When using the pillager bounding box, the width and length of the bounding box appear to only be 48 by 48, while the minecraft wiki says that the area is 72 by 72. Although I'm not sure if the wiki is outdated or this is a bug in the mod. 

![2022-09-01_19 29 41](https://user-images.githubusercontent.com/111377619/187977891-ee755f8b-67ef-4975-a089-845dc981360a.png)

![afbeelding_2022-09-01_193809995](https://user-images.githubusercontent.com/111377619/187977716-3e686f26-b154-4457-9b42-5b0fac29d025.png)

**Comment by maruohon** [comment]
The Structure Bounding Box renderer in MiniHUD shows exactly what the structure data in the world is. These are usually kind of "technical structures" related to the world generation structures/buildings. This is also what happens here.

For some structures, like Swamp Huts, Ocean Monuments and Nether Fortress, the world gen structure is directly also what affects mob spawns. But for Pillager Outposts, for whatever reason (maybe because the world gen structure would be too small for sufficient spawns?) the world gen structure is not the same where the pillagers spawn.

So basically this means that the renderer is correct - it shows the Pillager Outpost world gen structure. But in this case it's just not what you would want to see... I think I might add a separate renderer for the pillager spawn range at some point to cover this use case.

**Comment by tomosmc** [comment]
Ah, that explains. The spawning feature would be very nice, since there aren't any overlays or mods to do that I found, and it would be good for making a farm.
